syntax = "proto3";

package helloworld;

option csharp_namespace = "GrpcClientHelloWorld";

service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply);
}

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string message = 1;
}
